velox/exec/OutputBufferManager.cpp

```diff
@@ -183,4 +183,9 @@ bool OutputBufferManager::isOverutilized(const std::string& taskId) {
   return false;
 }
 
+OutputBufferStats OutputBufferManager::getStats(const std::string& taskId) {
+  auto buffer = getBuffer(taskId);
```

Comment:
I think this API should use getBufferIfExists and handle task missing or already finished and removed cases gracefully. Maybe it can return std::optional<OutputBufferStats>

Reply:
done.